src/main/scala/com/tapad/docker/ComposeFile.scala

```diff
@@ -183,7 +186,12 @@ trait ComposeFile extends SettingsHelper with ComposeCustomTagHelpers with Print
    * @param serviceKeys The Docker Compose Yaml representing a service
    * @return PortInfo collection for all defined ports
    */
-  def getPortInfo(serviceKeys: java.util.LinkedHashMap[String, Any]): List[PortInfo] = {
+  def getPortInfo(
+    serviceName: String,
+    serviceKeys: java.util.LinkedHashMap[String, Any],
+    useStatic: Boolean,
+    usedStaticPorts: scala.collection.mutable.Set[String]
```

Comment:
Does `getPortInfo` need to take `usedStaticPorts` as a parameter? I didn't see any callers of this function using the value.